docpad.coffee

```diff
@@ -112,9 +112,8 @@ module.exports =
 
     # List of Sponsors
     sponsors: [
-      # name: 'Eventick'
-      # logo: 'themes/yellow-swan/img/sponsor.png'
-      # url: 'http://eventick.com.br'
+      name: 'Seja um patrocinador'
+      logo: 'themes/yellow-swan/img/seja-um-patrocinador.jpg'
```

Comment:
No, not here.
It must be changed on `src/partials/section/sponsors.html.eco` (lines 11 ~ 15). And keep these comments.

Reply:
Make sure to test the changes with `npm run watch`